asmcli/asmcli.sh

```diff
@@ -1119,32 +1131,27 @@ EOF
   done <<EOF
 CLUSTER_LOCATION
 CLUSTER_NAME
-PROJECT_ID
```

Comment:
Without PROJECT_ID arg, how would the install command work with just cluster name and location?

We wanted 2 ways to install:
1) cluster name, location and project id
2) cluster kubeconfig and optional context (optional because current-context could suffice)

If #1 and #2 args are mixed, we do not proceed.

Reply:
`PROJECT_ID` is still required, it just got moved up to another block. I figured it was going to be required even for platforms where the kube context might not have the project information, so we might as well make it required all the time.

Reply:
Would the future need be for the environ id (may or may not be cluster's project) rather than the project id (cluster's project)?

Reply:
Yep, you're right. Fixed.